playbooks/roles/tarsnap/tasks/configure.yml

```diff
@@ -33,8 +33,11 @@
   cron:
     name: "Backup for this server, archive: {{ TARSNAP_ARCHIVE_NAME }}"
     job: "{{ TARSNAP_BACKUP_SCRIPT_LOCATION }} >> {{ TARSNAP_BACKUP_LOG_FILE }} 2>&1"
-    hour: "{{ TARSNAP_BACKUP_HOUR }}"
     minute: "{{ TARSNAP_BACKUP_MINUTE }}"
+    hour: "{{ TARSNAP_BACKUP_HOUR }}"
+    day: "{{ TARSNAP_BACKUP_DAY }}" 
+    month: "{{ TARSNAP_BACKUP_MONTH }}" 
+    week: "{{ TARSNAP_BACKUP_WEEK }}" 
```

Comment:
I don't see any option in the docs for `week`; it should be `weekday`. Consequently we should update the var to `TARSNAP_BACKUP_WEEKDAY` and remove the then redundant `# week day` comment next to it above.